### ISSUE TITLE
Highchart Height

### ISSUE THREAD

**Issue by sirvisuresh** [highcharts/highcharts#9391]
how to provide height of highchart of column type in terms of vh(view height) because it is taking height as percentage only and my second issue is whenever i am giving spacingBottom or marginBottom to legend which is aligned at top of the chart. it is reducing height of chart because it is relative.

**Comment by KacperMadej** [comment]
Hi @sirvisuresh

This is our bug tracker - for technical support contact [Highcharts support](https://www.highcharts.com/support)

P.S.
It's hard to follow you comment - please try to rewrite it when contacting support, so it will be clear what is the problem and add a live demo.
